packages/abi-gen/test/generated-test/generated-artifacts/AbiGenDummy.json

```diff
@@ -0,0 +1,175 @@
+{
+	"schemaVersion": "2.0.0",
+	"contractName": "AbiGenDummy",
+	"compilerOutput": {
```

Comment:
Did you check in the `generated-artifacts` folder on accident?  This change seems unrelated to the commit title ("prettier"), and I thought that the current setup has these files being re-generated on every test run (though btw I have doubts about the appropriateness of that approach...).

Reply:
gah. Yes these shouldn't be checked in. Thanks for the catch!